### PR TITLE
Feature/27/fix npm start

### PR DESCRIPTION
**Pull request by ChristofferTofteng**
Update README.me 
Add "start" script to package.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,33 +6,42 @@
...
 ## Deploy project
 
 In order to deploy to chat-bot to the server then please go through the following steps:
 
-1. Run the command `tsc`in the root of the project
-2. Connect to the server and navigate to the `headless-obyte` folder
-3. Upload all files in the project *except* the folder `node_modules` and `conf.js`
-4. Run `npm install` on the server within the `headless-obyte` folder
-5. Run the command `npm run start` to start the bot
+1. make a file called dbData.json in the root of the project containing mySQL credentials
```
Comment (joglr) [inline README.md:34]: Maybe write make with a captial M

File: README.md [hunk 2]
```diff
@@ -6,33 +6,42 @@
...
+2. Run the command `npm install` int the root of the project`
```
Comment (joglr) [inline README.md:35]: Accidental backtick

File: README.md [hunk 3]
```diff
@@ -6,33 +6,42 @@
...
+
+In case you want to use yarn make sure you have it installed by running the command `npm install --global yarn`
+
 ## Build project
 
 The PolloPollo chat-bot project is written in TypeScript and hence compilation is required in order to run the project. Therefore, to run the bot, please use the following steps:
 
-1. Run the command `tsc` in the root of the project
-2. Run the command `yarn start`/`npm run start` to run the bot
+1. Make a file called dbData.json in the root of the project containing mySQL credentials
```
Comment (joglr) [inline README.md:25]: Could you maybe give an inline example of what this file should look like?

e.g.
```json
{
  "thisIsHow": "you do"
}
````

**Review by nimrossum — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -6,33 +6,42 @@
...
 
 ### Typescript
 
 Furthermore typescript is required to compile the project to Node compatible JavaScript code.
 
 To install TypeScript run the command `npm install -g typescript` or `yarn global add typescript`
 
+### Yarn
+
+In case you want to use yarn make sure you have it installed by running the command `npm install --global yarn`
```
Comment (adamnegaard) [inline README.md:19]: Is Yarn not required to run the project?

File: src/config/apis.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-const basePath = "http://localhost:4000/api";
+const basePath = "http://localhost:5000/api";
```
Comment (adamnegaard) [inline src/config/apis.ts:1]: Possibly create a `.env` file like we did on the front-end?

File: README.md (hunk 4 above)
Comment (ChristofferTofteng) [reply README.md:19]: You can just use npm to call the scripts instead of yarn.

File: src/config/apis.ts (hunk 5 above)
Comment (ChristofferTofteng) [reply src/config/apis.ts:1]: Right now it isn't a task and it seemed easier and faster to just change a 4 to a 5

File: README.md (hunk 4 above)
Comment (ChristofferTofteng) [reply README.md]: Changed wording

**Review by nimrossum — APPROVED** [review]
Thanks for making the changes. Looks great!

**Review by ChristofferTofteng — COMMENTED** [review]
Thanks for formatting

**Review by adamnegaard — APPROVED** [review]
Looks good

**Pull request merged**